src/thor/bidirectional_astar.cc

```diff
@@ -136,23 +136,14 @@ void BidirectionalAStar::Init(const PointLL& origll, const PointLL& destll) {
   cost_threshold_ = std::numeric_limits<float>::max();
   iterations_threshold_ = std::numeric_limits<uint32_t>::max();
 
-  // Support for hierarchy transitions
-  hierarchy_limits_forward_ = costing_->GetHierarchyLimits();
-  hierarchy_limits_reverse_ = costing_->GetHierarchyLimits();
-  bool ignore_forward_limits =
-      std::all_of(hierarchy_limits_forward_.begin() + 1,
-                  hierarchy_limits_forward_.begin() + TileHierarchy::levels().size(),
-                  [](const HierarchyLimits& limits) {
-                    return limits.max_up_transitions == kUnlimitedTransitions;
-                  });
-  bool ignore_reverse_limits =
-      std::all_of(hierarchy_limits_reverse_.begin() + 1,
-                  hierarchy_limits_reverse_.begin() + TileHierarchy::levels().size(),
-                  [](const HierarchyLimits& limits) {
-                    return limits.max_up_transitions == kUnlimitedTransitions;
-                  });
-  // Set this flag to 'true' if we can expand edges at all hierarchy levels without limits
-  ignore_hierarchy_limits_ = ignore_forward_limits && ignore_reverse_limits;
+  auto& hierarchy_limits = costing_->GetHierarchyLimits();
+  ignore_hierarchy_limits_ = std::all_of(hierarchy_limits.begin() + 1,
+                                         hierarchy_limits.begin() + TileHierarchy::levels().size(),
+                                         [](const HierarchyLimits& limits) {
+                                           return limits.max_up_transitions == kUnlimitedTransitions;
+                                         });
+  hierarchy_limits_forward_ = hierarchy_limits;
+  hierarchy_limits_reverse_ = hierarchy_limits;
```

Comment:
`hierarchy_limits_forward_` and `hierarchy_limits_reverse_` are the exact same at this point, enough to do this whole thing once

Reply:
just wondering why are you resolving these? if they add context to the pr review its better to see them